sql/updates/README.md

```diff
@@ -58,3 +58,45 @@ Note that modfiles that contain no changes need not exist as a
 file. Transition modfiles must, however, be listed in the
 `CMakeLists.txt` file in the parent directory for an update script to
 be built for that version.
+
+## Extension downgrades
```

Comment:
I don't think this readme is good enough.

Let's say I'm working on a new feature. I wrote an upgrade script and, as I understand from the commit message, apparently there is a way to automatically generate a downgrade script. I would look for a step-by-step instructions on what commands to execute. Does "it is necessary to execute CMake" mean that execution cmake with any arguments will automatically generate a downgrade script? In any case, it's worth explicitly saying this.

Reply:
Added a comment explaining about the `GENERATE_DOWNGRADE_SCRIPT` flag and giving an example.